_includes/utilities/responsive-variants.html

```diff
@@ -14,7 +14,7 @@ <h4 class="font-lang-2xs margin-top-2 margin-bottom-05">Output</h4>
 }
 </pre>
 </div>
-  <p class="utility-text margin-top-4">Set which breakpoints are available to USWDS utilities with the <code>$theme_output_breakpoints</code> variable found in <code>_uswds-theme-utilities.scss</code>.</p>
+  <p class="utility-text margin-top-4">Set which breakpoints are available to USWDS utilities with the <code>$theme-utility-breakpoints</code> variable in your <a href="{{ site.baseurl }}/documentation/settings" class="">USWDS settings configuration</a>.</p> 
```

Comment:
@bonnieAcameron I wanted to run some content changes by you before making them. @amyleadem pointed out that some of these read as existing settings. I'm thinking sections like this could be changed to something like

> Set which breakpoints are available to USWDS utilities by setting the `theme-utility-breakpoints` variable in your USWDS settings configuration

How do you both feel about that?